theano/tensor/nnet/abstract_conv.py

```diff
@@ -1781,7 +1782,7 @@ def __init__(self,
                                                        filter_dilation=filter_dilation)
 
     # Update shape/height_width
-    def make_node(self, img, topgrad, shape):
+    def make_node(self, img, topgrad, shape, add_assert=True):
```

Comment:
Maybe call it `assert_shapes` or `add_assert_shape`, I think the name is too vague.
Also, it would need to be documented in the docstring, especially because it is a parameter of `make_node` that is not a symbolic input of the node.